models/comment_model.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"github.com/jinzhu/gorm"
+	"time"
+)
+
+type Comment struct {
```

Comment:
@breakingrobot Even if this is a bit explicit, still add comments to it for further GoDocs generation